src/main/java/org/prebid/cache/repository/CacheConfig.java

```diff
@@ -17,5 +19,8 @@
     private int timeoutMs;
     private long minExpiry;
     private long maxExpiry;
+    private List<String> secondaryIps;
```

Comment:
Proposed configuration approach seems a little too restraining. For example cache servers might run on different ports - it's impossible to utilize such setup with current configuration.

The most simple and straightforward approach might work better - have a list of URLs where requests need to be propagated, i.e. `List<String> secondaryUris` (assuming these URIs point to the root context of cache application)